galasa-ui/src/app/users/page.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+import { UserData, UsersAPIApi } from '@/generated/galasaapi';
+import { createAuthenticatedApiConfiguration } from '@/utils/api';
+import React, { useEffect } from 'react';
```

Comment:
useEffect isn't being used - can we remove unused imports? Same applies throughout this PR